### ISSUE TITLE
[Q] How to log a table of results with wandb?

### ISSUE THREAD

**Issue by sagnik** [wandb/wandb#4924]
I am trying to use wandb for a non DNN experiment.  Each `experiment` results in a dataframe like this: 

![image](https://user-images.githubusercontent.com/4563472/217369641-b64326d7-d4bd-4d12-b12e-0ab8466b4439.png)

typically class, x, y, z values will be str, and they can be the same (i.e., all x_is are the same), but at least one col will be different. In other words, (class, x, y, z) is a composite primary key.

This is how I am doing it rn:

```
class WandbLogger:
    def __init__(self, exp_name: str, exp_config: Dict):
        self._wandb = wandb.init(project=self.exp_name, config=self.exp_config, entity="my_entity")

    def done(self, result: pd.DataFrame):
        d = result.to_dict(orient="records")
        with self._wandb:
            for index, item in enumerate(d):
                item.update({"row": index})
                self._wandb.log(item)
        wandb.finish(exit_code=0)
```

I am expecting to see this when I have 1 experiment :

![image](https://user-images.githubusercontent.com/4563472/217371604-71a5f668-8c2b-44e5-9114-9f4236c9df30.png)

I am expecting to see this when I have 2 experiments that differ in 1 config param :

![image](https://user-images.githubusercontent.com/4563472/217371746-3d1c6344-0eab-43ab-a08f-ad449595295e.png)

For the first case, I am seeing this instead:

![image](https://user-images.githubusercontent.com/4563472/217371921-65ca049f-b15f-44f3-8333-c909a7f06ddf.png)

And the second case, I am seeing this:

![image](https://user-images.githubusercontent.com/4563472/217371980-d8144864-8d70-4d45-b558-4a54a06320bd.png)

In other words, `wandb` is only logging the last values in the iteration. How do I fix this?

**Comment by sagnik** [comment]
I did try using `wandb.Table` but that somehow didn't work, but I solved it by re-initing inside the loop. Thanks, closing for now.
